### PR TITLE
Add support for static web assets

### PR DESCRIPTION
**Pull request by danada**
Probably preferable to serve things like social images from a public gcloud bucket or something but it's harder to commit changes to the source. Maybe adding assets to a gcloud bucket with terraform is a better option. What do you think?

### CODE REVIEW

**Comment by curioswitch-role** [comment]
Build succeded. If you have approval, you're ready to merge!

Logs:
https://console.cloud.google.com/gcr/builds/e4e0f758-06ea-4b9a-897a-73195678042a?project=153942441948

**Comment by chokoswitch** [comment]
I think it's fine to serve assets from your frontend server, it should be using GKE's Ingress CDN

Having a folder that's directly copied seems useful so I don't have a problem with this change, with or without any other ones. I'm thinking of other alternatives too though.

- I think it's generally possible to override an output filename using `!` type manual configuration when loading a file with an `import` or `require` statement. You can require files even if you don't use them to make them go through the loader pipeline as usual, and by overriding the filename should be able to get one without a hash.

- For images, this could be especially nice since it goes through the optimization pipeline

- It's a bit annoying to both add a file and require it in code

We'll go ahead and merge this change but it'd probably be good to consider if an alternative using a loader with manual configuration could work.

File: common/web/base-web/src/dev/webpack/prod.ts [hunk 1]
```diff
@@ -90,6 +91,14 @@ const plugins = [
     statsFilename: 'iconstats.json',
   }),
 
+  new CopyWebpackPlugin([
```
Comment (chokoswitch) [inline common/web/base-web/src/dev/webpack/prod.ts]: Let's put this in `base.ts` - it's worth being able to confirm the assets with the dev server.

**Comment by danada** [comment]
Will all parsers/crawlers (like facebook, twitter, etc) wait for the JS and parse the correct `meta` tags provided in `<Helmet>`? I'm worried that they'll only look at index.html and I'm not sure if we can import anything there.

**Comment by chokoswitch** [comment]
By importing, I mean using `require('social_image.png!filename=foo.png')` in `app.ts` or something without actually using it - if you require something without using it, it still gets copied to the output folder. I haven't looked up the exact syntax for overriding the filename with a `!` but think it should be possible.

**Comment by chokoswitch** [comment]
Also - Helmet tags should get parsed OK anyways due to prerendering. Did prerendering stop working?

**Comment by danada** [comment]
Yeah, we ran into some issues with it after updating to MUI4 and the latest styled components. Haven't had much time to look at it yet, but will probably start investigating it this week or next.

**Comment by curioswitch-role** [comment]
Build succeded. If you have approval, you're ready to merge!

Logs:
https://console.cloud.google.com/gcr/builds/9fee94a0-50e5-4064-8444-dcfdca5034d8?project=153942441948

**Pull request merged**